docs/HowTo/Troubleshoot/Troubleshooting.md

```diff
@@ -105,7 +105,7 @@ If all recent attestations are marked as missed, check the following:
 * **Is the beacon node still syncing?**
 
     Validators can only attest when the beacon node is in sync. If you see lines similar to,
-    `teku-event-log | Sync Event  *** Current slot: 239418, Head slot: 123456`, then the node is
+    `teku-event-log | Syncing  *** Target slot: 2017218, Head slot: 123456, Remaining slots: 2015500, Connecting peers: 12`, then the node is
```

Comment:
if the '***' moves, this would be a bug in our output, im pretty sure it should stay in alignment...